judges/cuda/judge.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
+# vim:ts=4:sts=4:sw=4:expandtab
+import os, sys
+if __name__ == '__main__':
+    sys.path.insert(0, os.path.join(os.path.dirname(os.path.abspath(__file__)), 'kolejka-judge'))
+    from kolejka.judge import main
+    main(__file__)
+from kolejka.judge.commands import *
+from kolejka.judge.parse import *
+from kolejka.judge.tasks import *
+
+def judge(args):
+    tool_time = parse_time('60s')
+    prepare_time = parse_time('5s')
+    source_size_limit = parse_memory(args.test.get('source_size', '100K'))
+    binary_size_limit = parse_memory(args.test.get('binary_size', '10M'))
+    compile_time = parse_time(args.test.get('compile_time', '30s'))
+    cpp_standard = args.test.get('cpp_standard', 'c++17')
+    time_limit = parse_time(args.test.get('time', '10s'))
+    memory_limit = parse_memory(args.test.get('memory', '1G'))
+    time_gpu_limit = parse_time(args.test.get('time_gpu', '10s'))
+    memory_gpu_limit = parse_memory(args.test.get('memory_gpu', '1G'))
```

Comment:
Remove, or switch to metrics?

Reply:
Is it about `time_gpu_limit` right? The line selected is with the `memory_gpu_limit` but I think it should be line above?